integration/rootless/rootless_test.sh

```diff
@@ -27,19 +27,35 @@ setup() {
 
 cleanup() {
 	sudo sed -i -e 's/^.*\(rootless\)/# \1/g' /usr/share/defaults/kata-containers/configuration.toml
-	sudo crictl stopp "$pod_id" || true
-	sudo crictl rmp "$pod_id" || true
+	sudo crictl stopp "$pod_id" &>/dev/null || true
+	sudo crictl rmp "$pod_id" &>/dev/null || true
 }
 
 run() {
 	pod_id="$(sudo crictl runp -r kata "${dir_path}/rootless-pod.json")"
 	waitForProcess 15 3 "sudo crictl inspectp "$pod_id" | jq '.status.state' | grep 'SANDBOX_READY'"
+
+	# There must be created a temporary user.
+	local kata_uid="$(grep -e '^kata-[0-9]\+' /etc/passwd | cut -d: -f3)"
+	local kata_gid="$(grep -e '^kata-[0-9]\+' /etc/passwd | cut -d: -f4)"
+	[[ -n "$kata_uid" && -n "$kata_gid" ]]
+
+	# Assert the QEMU process is owned by that user.
+	ps -u "$kata_uid" | grep "qemu" &>/dev/null || \
```

Comment:
should we validate qemu process exists first?

Reply:
I'm not sure. And if there is a running qemu process, it is hard to tell for sure it belongs to this test case. However your comment made me review the code and I found some flaws....gonna send a v2.